lib/galaxy/tools/deps/mulled/mulled_build_channel.py

```diff
@@ -42,13 +42,7 @@ def _fetch_repo_data(args):
         repo_data = "%s-repodata.json" % channel
     if not os.path.exists(repo_data):
         platform_tag = 'osx-64' if sys.platform == 'darwin' else 'linux-64'
-        subprocess.check_call([
-            'wget', '--quiet', 'https://conda.anaconda.org/%s/%s/repodata.json.bz2' % (channel, platform_tag),
-            '-O', '%s.bz2' % repo_data
-        ])
-        subprocess.check_call([
-            'bzip2', '-d', '%s.bz2' % repo_data
-        ])
+        check_output("wget --quiet https://conda.anaconda.org/%s/%s/repodata.json.bz2 -O '%s.bz2' && bzip2 -d '%s.bz2'" % (channel, platform_tag, repo_data, repo_data))
```

Comment:
Using a string instead of an array goes against the recent improvements we've done to increase security, see warning box in https://docs.python.org/2.7/library/subprocess.html#frequently-used-arguments .

This is probably also broken because "If passing a single string, either shell must be True or else the string must simply name the program to be executed without specifying any arguments."

Reply:
That's actually a bad merge, see commit 19d03edc7ff8384a5360d71365076a2594fb0b8b